india_stocks_api/brokers/fivepaisa.py

```
@@ -0,0 +1,450 @@
+        return {"status": "error", "message": "GTT not yet implemented for 5paisa"}
+
+    def modify_gtt(self, id: int, instrument: Equity | Future | Option, 
+                  quantity: int, trigger_price: float, price: float) -> dict:
+        # TODO: Implement GTT modify for 5paisa if available
+        return {"status": "error", "message": "GTT not yet implemented for 5paisa"}
+
+    def cancel_gtt(self, id: int, instrument: Equity | Future | Option) -> dict:
+        # TODO: Implement GTT cancel for 5paisa if available
+        return {"status": "error", "message": "GTT not yet implemented for 5paisa"}
+
+    def get_gtt_list(self, status: list = ["FOR_SETTLEMENT", "CANCELLED", "TRIGGERED"]) -> list:
```

Ruff (0.14.14)

Unused method argument: `status`

(ARG002)